app/javascript/components/pages/ApartmentIndex.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+// import "@testing-library/jest-dom"
```

Comment:
Need additional testing in which case this commented out line will be useful. Need to check for certain content. ie What are you expecting to show up on the page?